### DISCUSSION TITLE
By dropping IE, there are better and low-cost-chances to support RTL

### DISCUSSION THREAD

**Discussion by themojilla** [element-plus/element-plus#3347] [Ideas]
Thank you for all your efforts on OSS.

As we know, the IE is already [dropped](https://github.com/vuejs/rfcs/discussions/296) in Vue3. By taking into the account the fact that IE is the main obstacle for using modern CSS, I think now we are ready to embrace the following rules and have a built-in RTL support:

- [padding-inline-start](https://caniuse.com/mdn-css_properties_padding-inline-start), [padding-inline-end](https://caniuse.com/mdn-css_properties_padding-inline-end) and for margins as-well
- [text-align-start/end](https://caniuse.com/mdn-css_properties_text-align_flow_relative_values_start_and_end), 

Recently, I wrote a custom `RTL.scss` to make things also work in RTL direction. The above rules almost do the trick for all components, for things like positions we still could take advantage of a SCSS RTL function in theme-chalk.

Beside the time, which plays a crucial role in this manner, do you think there are any drawbacks here?

Thanks.

**Comment by iamkun** [comment]
That looks cool to me. 

As we are not really familiar with RTL, and have no experience to implement this.

It will be great if you are interested to introduce RTL to Element Plus with us.

We can open a new RFC discussion to deal with it.
